src/manifest.json

```diff
@@ -13,5 +13,9 @@
     "48": "icon/icon-48.png",
     "128": "icon/icon-128.png",
     "512": "icon/icon-512.png"
-  }
+  },
+  "background": {
+    "service_worker": "sw.js"
+  },
+  "offline_enabled": true
```

Comment:
What does offline_enabled do? Got a link to documentaion?

Reply:
Defaults to true most of the time.

Here's the [doc](https://developer.chrome.com/docs/extensions/mv2/manifest/offline_enabled/)